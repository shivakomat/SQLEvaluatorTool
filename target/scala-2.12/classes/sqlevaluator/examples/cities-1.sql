-- Big cities in Japan
SELECT name, population
FROM cities
WHERE country = "Japan" AND population > 8000
