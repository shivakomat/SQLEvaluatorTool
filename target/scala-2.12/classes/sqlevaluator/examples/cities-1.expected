[
    [["name", "str"], ["population", "int"]],
    ["Tokyo", 13513],
    ["Kanagawa", 9127],
    ["Osaka", 8838]
]
